core_lib/src/structure/layercamera.cpp

```diff
@@ -33,6 +36,14 @@ LayerCamera::LayerCamera(Object* object) : Layer(object, Layer::CAMERA)
         mFieldH = 600;
     }
     viewRect = QRect(QPoint(-mFieldW / 2, -mFieldH / 2), QSize(mFieldW, mFieldH));
+/*
+    // to make sure it works for older .pclx files
+    for (int i = firstKeyFramePosition(); i <= getMaxKeyFramePosition(); i = getNextKeyFramePosition(i))
+    {
+        Camera* cam = getCameraAtFrame(i);
+        cam->setEasingType(cam->getEasingType());
+    }
+*/
```

Comment:
Looks neat! I don’t have the time to do a proper review rn, but I noticed this while skimming through the changes: Please try not to commit commented-out code unless there’s a good reason for it. And even then, please document that reason. :)

Reply:
Yeah, that's a blunder. As you can see, I caught an unused include, when doing the small improvement, and I think I've removed all the qDebug-messages I had in the code, but a review can reveal that.

Reply:
Removed dead code.